tests/devices/qutrit_mixed/test_qutrit_mixed_preprocessing.py

```diff
@@ -126,6 +126,7 @@ def test_measurement_is_swapped_out(self, mp_fn, mp_cls, shots):
             (qml.Snapshot(), True),
             (qml.TRX(1.1, 0), True),
             (qml.QutritDepolarizingChannel(0.4, 0), True),
+            (qml.QutritAmplitudeDamping(0.1, 0.2, 0.12, 0), True),
```

Comment:
Missed in the ampdamp channel? (Maybe worth checking other relevant test files)

Reply:
I think I just moved it in the merge so that TritFlip was bellow.

Reply:
Also, this is the only test setup currently, in the future I plan to add more in a future PR.